recipes/sandpyper/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "sandpyper" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 78392b70bd637a4ba2a42ce1268d4ac3aff09ff7ad677b15c9010f731c5aaa93
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+
+    - pip
+    - python >=3.8
+
+  run:
+    - fuzzywuzzy
+    - geopandas ==0.8.2
+    - matplotlib ==3.3.4
+    - numpy ==1.20.1
+    - pandas ==1.2.2
+    - pooch ==1.4.0
+    - pysal ==2.1
+    - python >=3.8
+    - rasterio ==1.2.0
+    - richdem ==0.3.4
+    - scikit-image ==0.18.1
+    - scikit-learn ==0.24.1
+    - scipy ==1.6.0
+    - seaborn ==0.11.1
+    - tqdm ==4.56.2
```

Comment:
Pinning strictly like this is generally a bad idea and causes a hard to debug mess once you hit conflicts (imagine if seaborn had `matplotlib==3.3.4`).

As you maintain upstream I'd recommend only having minimum pins. If you really want to pin things then they should be much more relaxed.

Reply:
Yes thanks for the suggestion @chrisburr . I update the code as soon as possible.